### PR TITLE
Apply content & formatting updates to Health Care App Veteran Info page

### PR DESCRIPTION
**Pull request by longmd**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

This PR applies some content formatting to the personal information page of the Health Care Application. There are also some minor content tweaks in here as well.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#81103

## Testing done

- **For authenticated users:**
- Sign in, navigate to the Intro page of the application and click one of the start buttons
- Review the formatting of the first page that appears
- **For guest users:**
- Navigate to the intro page of the application and click the start button at the bottom of the page that allows you to fill out the form as a guest
- Fill in the Veteran info on the identity verification page
- Review the formatting of the next page that appears

## Screenshots

**Authenticated user:**

| Before | After |
| ----------- | ------------------- |
| Screenshot 2024-08-09 at 15 38 31 | Screenshot 2024-08-09 at 14 57 05

**Guest user:**

| Before | After |
| ----------- | ------------------- |
| Screenshot 2024-08-09 at 15 40 41 | Screenshot 2024-08-09 at 14 56 28 |

## Acceptance criteria

 - Content formatting conveys appropriate meaning to the Veteran

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js [hunk 1]
```diff
@@ -56,86 +56,58 @@ describe('hca VeteranInformation', () => {
       dispatch: () => {},
     },
   });
+  const subject = ({ mockStore, props }) => {
```
Comment (coope93) [inline src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js:59]: I love how much code using these subjects with the selectors simplifies 👍

File: src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx [hunk 2]
```diff
@@ -7,58 +7,66 @@ import { normalizeFullName } from '../../utils/helpers';
...
+        </ul>
+      </va-card>
       <p>
         <strong>Note:</strong> If you need to update your personal information,
         call our VA benefits hotline at{' '}
         <va-telephone contact={CONTACTS.VA_BENEFITS} /> (
         <va-telephone contact={CONTACTS[711]} tty />
-        ), Monday through Friday, 8:00 a.m. to 9:00 p.m.{' '}
+        ). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m.{' '}
         <dfn>
```
Comment (coope93) [inline src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx:61]: Could this be replaced with our `Abbr` component?

File: src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import { maskSSN } from '../../../../utils/helpers/general';
+
+describe('hca `maskSSN` method', () => {
```
Comment (coope93) [inline src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js:5]: Any concerns around passing a string that is 4 characters or less? I know we probably won't use it except for when we have a full ssn, but what would we expect the `maskSSN` to return for example if it was passed `333`?

**Review by coope93 — APPROVED** [review]
Nice work on this! I left a few comments but nothing is a blocker.

File: src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx (hunk 2 above)
Comment (longmd) [reply src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx:61]: Yes, definitely! I want to do that, as its own PR, as a global find and replace in the application as a whole.

File: src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js (hunk 1 above)
Comment (longmd) [reply src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js:59]: Same here!! I didn't quite understand when I first watched you implement it.. but it really does clean these tests up a ton!

File: src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js (hunk 3 above)
Comment (longmd) [reply src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js:5]: We've never had one come across short on character count or formatted incorrectly. It's always been an all-or-nothing-type thing. I supposed we could try to pass one that isn't formatted correctly and see what happens.

**Pull request merged**